lib/tbd/psi.rb

```diff
@@ -394,12 +423,14 @@ def initialize
         parapet:      0.056000, # "parapet" edge
         roof:         0.150000, # (non-parapet) "roof edge"
         fenestration: 0.260000, # "wall to vertical fenestration intersection"
+        door:         0.000000, # (unspecified, defaults to 0)
+        skylight:     0.000000, # (unspecified, defaults to 0)
         spandrel:     0.000000, # (unspecified, defaults to 0)
-        corner:       0.000000, # (unspecified, defaults to 0)
-        balcony:      0.000000, # "intermediate floor balcony/overhang" edge
-        balconysill:  0.260001, # "intermediate floor balcony" edge (when sill)
-        party:        0.000000, # (unspecified, defaults to 0)
-        grade:        0.000000, # (unspecified, defaults to 0)
+        corner:       0.000001, # (unspecified, defaults to 0)
+        balcony:      0.582000, # same as :rimjoist
+        balconysill:  0.582000, # same as :rimjoist
```

Comment:
Oddly, 90.1 (A10) doesn't hold **:balcony** or **:balconysill** PSI factors for "wood-framed" construction (while it does for other construction choices). So is a "_fenestrated door_" sill on a balcony (in a wood-framed MURB):

- unregulated?
- a "wall to _vertical fenestration_ intersection"?
- an "intermediate floor to wall intersection"?

Postulating the most stringent case ("intermediate floor to wall intersection") for now ...